flexget/components/sites/sites/newznab.py

```diff
@@ -123,6 +127,11 @@ def do_search_movie(self, arg_entry, task, config=None):
         url = config['url'] + '&imdbid=' + imdb_id
         return self.fill_entries_for_url(url, task)
 
+    def do_search_all(self, arg_entry, task, config=None):
+        log.info('Searching for %s' % arg_entry['title'])
```

Comment:
please replace the `%s` with a `,` to let the logger do its own string interpolation